profydev/prolog-app-jaclynmariefrench
# Fix navigation support button

### Expected behavior
Clicking the “Support” button in the sidebar navigation should open the user’s email app.

### Past behavior
Clicking the support button opens an alert. There’s no further interaction or information available.
![image](https://github.com/profydev/prolog-app-jaclynmariefrench/assets/77642588/7cfdbdbc-124d-4428-bb62-7ca44ace666e)

### Test
1. Navigate to /dashboard
2. Verify that when clicking 'Support' on nav bar that your email pops up with the recipient being “support@prolog-app.com”
and subject line should be pre-filled with “Support Request: “. 
3. Run npm run cypress
4. Look at E2Es text navigation
5. Verify that all tests have passed and look at the navigation links tests to verify the support test has passed.
